internal/util/log/log_utils_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2019 ceph-csi authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package log
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestGzipLogFile(t *testing.T) {
+	t.Parallel()
+	logFile, err := ioutil.TempFile("/tmp", "rbd-*.log")
```

Comment:
instead of hardcoding `/tmp`, consider using [`t.TempDir()`](https://pkg.go.dev/testing#T.TempDir)